README.md

```diff
@@ -115,7 +115,7 @@ Event::log("my_event_name");
    ```
 4. In den Einstellungen das tracken aktivieren.
 5. Das Beipiel erzeugt dann eine solche Tabelle:  
-   ![Beispiel1](./preview/6.png "Beispiel1")
+   ![Beispiel1](https://raw.githubusercontent.com/andileni/statistics/main/preview/6.png "Beispiel1")
```

Comment:
Sehe ich das richtig, dass jetzt die README.md nicht mehr offline angezeigt werden kann?

Reply:
@alxndr-w ich meine das ist doch in den meisten Readme so dass hier auf die Online-Bilder verlinkt ist.